### PR TITLE
JBIDE-28571, update to itests, updating EAP 7.4 templates

### PR DESCRIPTION
**Pull request by odockal**
# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by odockal** [comment]
@olkornii Please review.

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/CreateResourcesTest.java [hunk 1]
```diff
@@ -71,15 +71,15 @@ public void prepareTestEnvironment() {
 		explorer.getOpenShift3Connection(connectionReq.getConnection()).createNewProject(testProject);
 	}
 
-	@Test
+//	@Test
```
Comment (olkornii) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/CreateResourcesTest.java:74]: Some disabled tests, check if its ok pls

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/CreateResourcesTest.java [hunk 2]
```diff
@@ -71,15 +71,15 @@ public void prepareTestEnvironment() {
...
 	public void testOpenCreateResourceWizardViaContextMenuOfConnection() {
 		explorer.getOpenShift3Connection(connectionReq.getConnection()).select();
 		new ContextMenuItem(OpenShiftLabel.ContextMenu.NEW_RESOURCE).select();
 
 		assertResourceShellIsAvailable();
 	}
 
-	@Test
+//	@Test
```
Comment (olkornii) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/CreateResourcesTest.java:82]: Some disabled tests, check if its ok pls

File: itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/CRCAllTestsSuite.java [hunk 3]
```diff
@@ -26,8 +26,8 @@
  */
 @RunWith(RedDeerSuite.class)
 @Suite.SuiteClasses({
-	CRCRuntimeDetectionTest.class,
-	CRCDownloadRuntimeTest.class,
+//	CRCRuntimeDetectionTest.class,
```
Comment (olkornii) [inline itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/CRCAllTestsSuite.java:29]: Some disabled tests, check if its ok pls

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationFromTemplateTest.java [hunk 4]
```diff
@@ -202,7 +205,7 @@ public void createApplicationFromTemplateProvidedByURL() {
 		completeApplicationCreationAndVerify(helloworldProject, 2);
 	}
 
-	@Test
+//	@Test
```
Comment (olkornii) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationFromTemplateTest.java:208]: Some disabled tests, check if its ok pls

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationFromTemplateTest.java [hunk 5]
```diff
@@ -186,10 +189,10 @@ public void createApplicationFromLocalFileSystemTemplate() {
 //		assertTrue("Defined resource button should be enabled",
 //				new PushButton(OpenShiftLabel.Button.DEFINED_RESOURCES).isEnabled());
 
-		completeApplicationCreationAndVerify(helloworldProject, 2);
+		completeApplicationCreationAndVerify(kitchensinkProject, 2);
 	}
 
-	@Test
+//	@Test
```
Comment (olkornii) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationFromTemplateTest.java:195]: Some disabled tests, check if its ok pls

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/InteligentDeleteResourceTest.java [hunk 6]
```diff
@@ -154,7 +154,7 @@ public void testDeleteIRoute() {
 	}
 	
 	@Test
-	public void testDeleteEReplicationController() {
+	public void testDeleteeReplicationController() {
```
Comment (olkornii) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/InteligentDeleteResourceTest.java:157]: "...Deletee", mb typo?

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/odo/DebugVertxDevfileComponentODOTest.java [hunk 7]
```diff
@@ -45,7 +45,7 @@
 @RunWith(RedDeerSuite.class)
 @RequiredODOConnection
 @CleanODOConnection
-@RequiredODOProject(name="test-project7")
+@RequiredODOProject(name="test-project5", cleanup=true)
```
Comment (olkornii) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/odo/DebugVertxDevfileComponentODOTest.java:48]: "name = "test-project5", cleanup = true" looks better

**Review by olkornii — CHANGES_REQUESTED** [review]

**Comment by odockal** [comment]
@olkornii ready.

**Review by olkornii — APPROVED** [review]
LGTM

**Review by olkornii — APPROVED** [review]

**Review by olkornii — APPROVED** [review]

**Review by sbouchet — APPROVED** [review]
LGTM

**Pull request merged**